notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>args=[ ...., ... ]</p><p> </p><p>should be</p><p> </p><p>args=[arg1,</p><p>           arg2,</p><p>           ...</p><p>          ]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>